File: 백준/Bronze/11721. 열 개씩 끊어 출력하기/열 개씩 끊어 출력하기.java
import java.util.*;

public class Main {
    public static void main(String[] args) {
        Scanner sc=new Scanner(System.in);

        String str1=sc.nextLine();
        int cnt=0;
        String word="";

        for(int i=0;i<str1.length();i++){
            word+=str1.charAt(i);
            cnt+=1;
            //word+=str1[i];

            if(cnt==10){
                System.out.println(word);
                cnt=0;
                word="";
            }
            else if(i==str1.length()-1){
                System.out.println(word);
            }
        }



    }
}

File: 백준/Bronze/11721. 열 개씩 끊어 출력하기/열 개씩 끊어 출력하기.py
str1=input()
cnt=0
word=''

for i in range(len(str1)):
    word+=str1[i]
    cnt+=1
    if(cnt==10):
        print(word)
        cnt=0
        word=''
    elif(i==len(str1)-1):
        print(word)